rest/model/patch_test.go

```diff
@@ -101,6 +103,32 @@ func TestAPIPatch(t *testing.T) {
 	assert.Len(a.VariantsTasks[0].Tasks, 1)
 }
 
+func TestAPIPatchIncludeBranch(t *testing.T) {
+	// TODO DEVPROD-1682: remove, since this tests backwards compatibility behavior.
```

Comment:
Is this TODO still relevant? It seems to point to an unrelated ticket.

Reply:
oops this is just a typo, should be DEVPROD-16824